flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/source/connection/MySqlConnectionWithJdbcProperties.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ververica.cdc.connectors.mysql.source.connection;
+
+import io.debezium.config.Configuration;
+import io.debezium.connector.mysql.MySqlConnection;
+import io.debezium.connector.mysql.MySqlConnectorConfig;
+import io.debezium.jdbc.JdbcConnection;
+
+import java.util.Properties;
+
+/** {@link MySqlConnection} extension to be used with MySQL Server. */
+public class MySqlConnectionWithJdbcProperties extends MySqlConnection {
```

Comment:
(1) `MySqlConnectionWithJdbcProperties` can not visit `MySqlConnection`'s default package level methods due to the two classes are lying different package.